packages/cli-dashboard/src/app.tsx

```diff
@@ -180,13 +186,18 @@ const App = () => {
         landingPageCookies={landingPageCookies}
         cookies={cookies}
         technologies={technologies}
+        completeJson={completeJsonReport}
       />
     );
   }
 
   return (
     <div className="w-full h-screen flex">
-      <SiteReport cookies={cookies} technologies={technologies} />
+      <SiteReport
```

Comment:
If complete json report is being passed then i dont think there is a need for passing other 2 props.